src/odemis/gui/cont/correlation.py

```
@@ -261,13 +262,13 @@ def _update_localization_fibsem_tab_streams(self, streams: list) -> None:
         """
         if self.localization_tab is None:
             try:
-                self.localization_tab: "LocalizationTab" = self._main_data_model.getTabByName("cryosecom-localization")
+                self.localization_tab: "LocalizationTab" = self._main_data_model.getTabByName(TabName.CRYOSECOM_LOCALIZATION)
```

Ruff (0.15.9)

[error] Undefined name `LocalizationTab`

(F821)